src/containers/ImportRecordingsInstructions.tsx

```diff
@@ -17,6 +18,10 @@ const ImportRecordingsInstructions: FunctionComponent<Props> = () => {
                 <CopyToClipboardButton text={import_example_simulated_recording_py} />
                 <Markdown source={mdWrapPy(import_example_simulated_recording_py)} />
             </Expandable>
+            <Expandable label="Import SpikeForest recordings">
```

Comment:
Now there are two example scripts for importing recordings. (a) simulation or (b) from spikeforest